style.css

```diff
@@ -739,3 +835,101 @@ input::placeholder {
     justify-content: center;
   }
 }
+
+/* Popup */
+.popCard {
+  display: none;
+  width: 100%;
+  height: 100%;
+  position: fixed;
+  overflow: auto;
+  top: 0;
+  left: 0;
+  z-index: 1;
+  background-color: #c1c7d0(0, 0, 0, 0.9);
+  backdrop-filter: blur(15px);
+}
+
+.modal-body {
+  background-color: #fefefe;
+  margin: 5% auto;
+  padding: 1.25rem;
+  width: 80%;
+  border-radius: 16px;
+  border: 1px solid #ebecf0;
+  box-shadow: 0.1px 48px 48px rgba(37, 47, 137, 0.08);
+}
```

Comment:
- [ ] As mentioned previously please make sure the popup on the mobile version doesn't create an overflow as you can see from the below video it's not fitting proportionally to a screen of size `375px` which is the ideal mobile screen width so please carefully adjust this.
- Your design:
  ![firefox_p7i54SsgUa](https://github.com/katarighe/my-portfolio/assets/104968236/04aaf39b-5a7b-4ed1-845b-e2c88663d261)
- Figma design:
  ![image](https://github.com/katarighe/my-portfolio/assets/104968236/03800b80-1318-47db-af24-b857f6e8f5ee)